packages/frontend/src/components/screens/AccountSetupScreen.tsx

```diff
@@ -32,25 +34,31 @@ export default function AccountSetupScreen({
 
   const [credentials, setCredentials] =
     useState<Credentials>(defaultCredentials())
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const settingsStore = useSettingsStore()[0]!
 
-  const onClickLogin = useCallback(
-    () =>
-      openDialog(ConfigureProgressDialog, {
-        credentials,
-        onSuccess: () => {
-          selectAccount(accountId)
-        },
-        onFail: (error: string) =>
-          setPromptDialogId(
-            openDialog(AlertDialog, {
-              message: error,
-              cb: () => setPromptDialogId(null),
-            })
-          ),
-        proxyUpdated: credentials.proxyEnabled,
-      }),
-    [accountId, openDialog, selectAccount, credentials]
-  )
+  const onClickLogin = useCallback(() => {
+    openDialog(ConfigureProgressDialog, {
+      credentials,
+      onSuccess: async () => {
+        const profileComplete = await selectAccount(accountId)
```

Comment:
The same code could also go to https://github.com/deltachat/deltachat-desktop/blob/beebd9e4e4b8a43555368ef0fe810110bdbe7e77/packages/frontend/src/components/AccountListSidebar/AccountListSidebar.tsx#L65
Then we would show the EditProfile dialog on any account change if the selected account has no displayname 
🤔